packages/templates/clients/websocket/python/README.md

```diff
@@ -3,18 +3,53 @@
 
 To run the Slack Client example, follow the steps above but with 2 exceptions:
 - Use `example-slack.py` instead of `example.py`.
-- You need to pass environment variables with proper authorization details. Your command must look like this: `TICKET={provide secret info} APP_ID={provide id of the slack app} python example-slack.py`
+- You need to pass environment variables with proper authorization details. Your command must look like this: `TICKET=1d967f38-ccff-44f6-adec-9616eec9c4b7 APP_ID=00dfdcccb53a2645dd3f1773fcb10fa7b0a598cf333a990a9db12375ef1865dd python example-slack.py`
 
-Instructions on how to create the Slack app and also obtain authorization is described in details in the [Slack AsyncAPI document](../test/__fixtures__/asyncapi-slack-client.yml).
\ No newline at end of file
+> Instructions on how to create the Slack app and also obtain authorization is described in details in the [Slack AsyncAPI document](../test/__fixtures__/asyncapi-slack-client.yml).
+
+You can use our AsyncAPI's credentials to access different set of events produced in AsyncAPI Slack workspace, in the `#generator` channel.
+
+1. Generate an access ticket with an application ID that will enable you to establish a websocket connection:
+    ```
+    curl --location --request POST 'https://slack.com/api/apps.connections.open' \
+    --header 'Authorization: Bearer xapp-1-A08NKKBFGBD-8768087393844-b85d0215e33ec13272e523e47d96b602d9dc5e57863e5b2df79d81b999da10bc'
+    ```
+
+    Example response with `ticket` and `app_id`:
+    ```
+    {"ok":true,"url":"wss:\/\/wss-primary.slack.com\/link\/?ticket=089a0c38-cdec-409f-99fa-0ca24e216ea4&app_id=00dfdcccb53a2645dd3f1773fcb10fa7b0a598cf333a990a9db12375ef1865dd"}
+    ```
+
+    You can take generated `url` and use with CLI websocket client like `websocat` (first remove excape backslashes):
+    ```
+    websocat "wss://wss-primary.slack.com/link/?ticket=089a0c38-cdec-409f-99fa-0ca24e216ea4&app_id=00dfdcccb53a2645dd3f1773fcb10fa7b0a598cf333a990a9db12375ef1865dd"
+    ```
+
+    But that is just for testing. The point is to use the generated Python client.
+
+1. Start the example that uses generated client. Examine events, and modify example as you want:
+    ```
+    TICKET=089a0c38-cdec-409f-99fa-0ca24e216ea4 APP_ID=00dfdcccb53a2645dd3f1773fcb10fa7b0a598cf333a990a9db12375ef1865dd python example-slack.py
+    ```
+    Successfully established connection will welcome you with below event:
+    ```
+    {"type":"hello","num_connections":1,"debug_info":{"host":"applink-3","build_number":118,"approximate_connection_time":18060},"connection_info":{"app_id":"A08NKKBFGBD"}}
+    ```
+    If you did not receive it, you probably connect with wrong credentials. Remember that generated `ticket` can be used only once to establish a websocket connection.
+
+### Security Concerns
+
+The `App-Level Token` that is shared in this readme can be used only to obtain the websocket ticket. This token has only `connections:write` scope.
```

Comment:
Is `connections:write` a good scope? I would understand `connections:read` but write seems too big a scope to me. Can you explain?

Reply:
Ok, discussed in PM.

Reply:
as discussed, it only gives rights to open connection